nservicebus/licensing/index_license-management_core_[8,].partial.md

```diff
@@ -53,3 +53,20 @@ NOTE: If `HKEY_LOCAL_MACHINE` is the chosen license location, and the operating
 NOTE: If the license is stored in `HKEY_CURRENT_USER`, NServiceBus processes must run as the user account used to add the license file to the registry in order to access the license.
 
 It is safe to ignore any warnings regarding empty strings.
+
+
+### Environment variable location
+
+Finally a license can also be specified via setting the `PARTICULARSOFTWARE_LICENSE` environment variable with the license text
+
+* Windows
+  ```powershell
+  $env:PARTICULARSOFTWARE_LICENSE = @"the license text goes here
+  and is a multiline string"@
+  ```
+* Linux/macOS
+  ```bash
+  export PARTICULARSOFTWARE_LICENSE=`cat ./license.xml`
```

Comment:
Should we for consistency with the windows part only show the approach below?

Reply:
well, the reason to show this one is because I don't know the syntax in PS to do the same

Reply:
If someone else know how to do that in PS, we can add that later on